test/mocks/router/mocks.h

```diff
@@ -310,17 +310,16 @@ class MockRouteConfigProviderManager : public RouteConfigProviderManager {
   MockRouteConfigProviderManager();
   ~MockRouteConfigProviderManager();
 
-  MOCK_METHOD3(getRdsRouteConfigProvider,
-               RouteConfigProviderSharedPtr(
+  MOCK_METHOD3(createRdsRouteConfigProvider,
+               RouteConfigProviderPtr(
                    const envoy::config::filter::network::http_connection_manager::v2::Rds& rds,
                    Server::Configuration::FactoryContext& factory_context,
                    const std::string& stat_prefix));
-  MOCK_METHOD2(
-      getStaticRouteConfigProvider,
-      RouteConfigProviderSharedPtr(const envoy::api::v2::RouteConfiguration& route_config,
-                                   Server::Configuration::FactoryContext& factory_context));
-  MOCK_METHOD0(getRdsRouteConfigProviders, std::vector<RouteConfigProviderSharedPtr>());
-  MOCK_METHOD0(getStaticRouteConfigProviders, std::vector<RouteConfigProviderSharedPtr>());
+  MOCK_METHOD2(createStaticRouteConfigProvider,
+               RouteConfigProviderPtr(const envoy::api::v2::RouteConfiguration& route_config,
+                                      Server::Configuration::FactoryContext& factory_context));
+  MOCK_METHOD0(getRdsRouteConfigProviders, std::vector<RouteConfigProvider*>());
```

Comment:
why these two functions are still in the mock?   I thought they are removed.

Reply:
done